package.json

```diff
@@ -53,15 +54,15 @@
     "prettier": "^1.19.1"
   },
   "resolutions": {
-    "sharp": "0.24.0"
+    "sharp": "0.29.3"
   },
   "keywords": [
     "gatsby"
   ],
   "license": "MIT",
   "scripts": {
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop --host=0.0.0.0",
```

Comment:
what is this for?

Reply:
Make it easier for me to check and test on mobile device on the local network